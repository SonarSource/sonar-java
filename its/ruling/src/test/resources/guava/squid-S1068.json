{
'com.google.guava:guava:src/com/google/common/xml/XmlEscapers.java':[
116,
],
}
