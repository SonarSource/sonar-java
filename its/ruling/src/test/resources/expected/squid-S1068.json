{
'project:com/sun/corba/se/impl/activation/ServerTableEntry.java':[
87,
],
'project:com/sun/corba/se/impl/activation/ServerTool.java':[
228,
],
'project:com/sun/corba/se/impl/copyobject/CopierManagerImpl.java':[
40,
],
'project:com/sun/corba/se/impl/corba/AnyImpl.java':[
115,
],
'project:com/sun/corba/se/impl/corba/ContextImpl.java':[
45,
],
'project:com/sun/corba/se/impl/corba/ContextListImpl.java':[
44,
],
'project:com/sun/corba/se/impl/corba/RequestImpl.java':[
88,
89,
90,
],
'project:com/sun/corba/se/impl/corba/ServerRequestImpl.java':[
61,
],
'project:com/sun/corba/se/impl/encoding/BufferManagerReadGrow.java':[
42,
],
'project:com/sun/corba/se/impl/encoding/CDRInputObject.java':[
65,
],
'project:com/sun/corba/se/impl/encoding/CDROutputObject.java':[
67,
],
'project:com/sun/corba/se/impl/encoding/CDROutputStream_1_0.java':[
118,
],
'project:com/sun/corba/se/impl/encoding/CodeSetConversion.java':[
698,
],
'project:com/sun/corba/se/impl/encoding/IDLJavaSerializationInputStream.java':[
81,
],
'project:com/sun/corba/se/impl/encoding/IDLJavaSerializationOutputStream.java':[
72,
],
'project:com/sun/corba/se/impl/encoding/TypeCodeInputStream.java':[
71,
],
'project:com/sun/corba/se/impl/interceptors/CodecFactoryImpl.java':[
50,
],
'project:com/sun/corba/se/impl/interceptors/InterceptorInvoker.java':[
60,
],
'project:com/sun/corba/se/impl/interceptors/ORBInitInfoImpl.java':[
92,
],
'project:com/sun/corba/se/impl/io/IIOPInputStream.java':[
181,
],
'project:com/sun/corba/se/impl/io/ValueHandlerImpl.java':[
123,
],
'project:com/sun/corba/se/impl/ior/IdentifiableFactoryFinderBase.java':[
46,
],
'project:com/sun/corba/se/impl/ior/ObjectKeyFactoryImpl.java':[
90,
],
'project:com/sun/corba/se/impl/ior/ObjectKeyTemplateBase.java':[
60,
],
'project:com/sun/corba/se/impl/ior/WireObjectKeyTemplate.java':[
56,
],
'project:com/sun/corba/se/impl/ior/iiop/IIOPAddressImpl.java':[
44,
],
'project:com/sun/corba/se/impl/ior/iiop/IIOPProfileImpl.java':[
109,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/PersistentBindingIterator.java':[
141,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/ServantManagerImpl.java':[
72,
],
'project:com/sun/corba/se/impl/oa/poa/Policies.java':[
54,
],
'project:com/sun/corba/se/impl/orb/DataCollectorBase.java':[
57,
],
'project:com/sun/corba/se/impl/orb/ORBDataParserImpl.java':[
58,
72,
73,
],
'project:com/sun/corba/se/impl/orb/ORBSingleton.java':[
139,
],
'project:com/sun/corba/se/impl/orbutil/CacheTable.java':[
57,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
1181,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamField.java':[
268,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryIdCache_1_3.java':[
42,
43,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryIdCache_1_3_1.java':[
36,
37,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3.java':[
87,
124,
127,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3_1.java':[
91,
122,
129,
132,
],
'project:com/sun/corba/se/impl/presentation/rmi/PresentationManagerImpl.java':[
65,
115,
],
'project:com/sun/corba/se/impl/protocol/BootstrapServerRequestDispatcher.java':[
67,
],
'project:com/sun/corba/se/impl/protocol/CorbaInvocationInfo.java':[
46,
],
'project:com/sun/corba/se/impl/protocol/CorbaServerRequestDispatcherImpl.java':[
98,
],
'project:com/sun/corba/se/impl/protocol/RequestDispatcherRegistryImpl.java':[
54,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateReplyMessage_1_0.java':[
55,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateReplyMessage_1_1.java':[
54,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateReplyMessage_1_2.java':[
60,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/ReplyMessage_1_0.java':[
58,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/ReplyMessage_1_1.java':[
58,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/ReplyMessage_1_2.java':[
62,
],
'project:com/sun/corba/se/impl/transport/CorbaResponseWaitingRoomImpl.java':[
69,
],
'project:com/sun/corba/se/impl/transport/ListenerThreadImpl.java':[
47,
],
'project:com/sun/corba/se/impl/transport/ReaderThreadImpl.java':[
46,
],
'project:com/sun/corba/se/impl/util/RepositoryId.java':[
85,
116,
123,
126,
],
'project:com/sun/corba/se/impl/util/RepositoryIdCache.java':[
42,
43,
],
'project:com/sun/corba/se/internal/CosNaming/BootstrapServer.java':[
51,
],
'project:com/sun/corba/se/spi/orbutil/fsm/FSMTest.java':[
159,
],
'project:com/sun/corba/se/spi/protocol/ForwardException.java':[
40,
],
'project:com/sun/corba/se/spi/servicecontext/ServiceContextRegistry.java':[
37,
],
'project:com/sun/imageio/plugins/bmp/BMPImageWriter.java':[
88,
],
'project:com/sun/imageio/plugins/png/PNGImageReader.java':[
165,
],
'project:com/sun/imageio/plugins/png/PNGImageWriter.java':[
339,
],
'project:com/sun/java/swing/plaf/motif/MotifIconFactory.java':[
57,
],
'project:com/sun/java/swing/plaf/windows/WindowsButtonUI.java':[
161,
],
'project:com/sun/java/swing/plaf/windows/WindowsLookAndFeel.java':[
107,
114,
114,
114,
114,
115,
2153,
],
'project:com/sun/jmx/remote/security/FileLoginModule.java':[
151,
],
'project:com/sun/jmx/snmp/IPAcl/Parser.java':[
1034,
],
'project:com/sun/jmx/snmp/SnmpPeer.java':[
109,
],
'project:com/sun/jmx/snmp/daemon/CommunicatorServer.java':[
230,
],
'project:com/sun/jmx/snmp/daemon/SnmpMibTree.java':[
266,
],
'project:com/sun/jmx/snmp/daemon/SnmpRequestHandler.java':[
1150,
],
'project:com/sun/jmx/snmp/internal/SnmpEngineImpl.java':[
109,
],
'project:com/sun/jmx/snmp/tasks/ThreadService.java':[
244,
245,
],
'project:com/sun/org/apache/bcel/internal/util/CodeHTML.java':[
73,
],
'project:com/sun/org/apache/bcel/internal/util/SyntheticRepository.java':[
89,
],
'project:com/sun/org/apache/xalan/internal/xsltc/cmdline/Transform.java':[
61,
65,
66,
68,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Mode.java':[
70,
105,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ParentLocationPath.java':[
50,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Sort.java':[
81,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Stylesheet.java':[
76,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/UnsupportedElement.java':[
46,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/UseAttributeSets.java':[
46,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XPathLexer.java':[
38,
163,
224,
257,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/CompareGenerator.java':[
46,
47,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/MethodGenerator.java':[
56,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/DOMAdapter.java':[
60,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/LoadDocument.java':[
51,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
90,
98,
102,
131,
151,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/StepIterator.java':[
59,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/Attributes.java':[
34,
35,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/OutputSettings.java':[
33,
34,
35,
36,
37,
38,
39,
40,
41,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2DOM.java':[
60,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SmartTransformerFactoryImpl.java':[
60,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/StAXEvent2SAX.java':[
76,
77,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/StAXStream2SAX.java':[
81,
82,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TrAXFilter.java':[
55,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java':[
99,
100,
101,
102,
106,
],
'project:com/sun/org/apache/xerces/internal/impl/XML11DTDScannerImpl.java':[
101,
104,
110,
111,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentFragmentScannerImpl.java':[
2524,
2525,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentScannerImpl.java':[
230,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
2748,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLStreamReaderImpl.java':[
118,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLVersionDetector.java':[
129,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/DTDGrammar.java':[
279,
285,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDProcessor.java':[
126,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
98,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/models/CMNode.java':[
186,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/models/DFAContentModel.java':[
209,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/util/Base64.java':[
47,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/XPath.java':[
49,
799,
1236,
1237,
1238,
1283,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/SchemaGrammar.java':[
258,
259,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaLoader.java':[
236,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
1107,
1111,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/CMBuilder.java':[
44,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSDFACM.java':[
168,
],
'project:com/sun/org/apache/xerces/internal/jaxp/JAXPValidatorComponent.java':[
107,
352,
],
'project:com/sun/org/apache/xerces/internal/jaxp/SAXParserFactoryImpl.java':[
64,
],
'project:com/sun/org/apache/xerces/internal/jaxp/datatype/DurationImpl.java':[
112,
129,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractSAXParser.java':[
213,
],
'project:com/sun/org/apache/xerces/internal/parsers/XMLGrammarPreparser.java':[
94,
],
'project:com/sun/org/apache/xerces/internal/util/URI.java':[
240,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XPointerElementHandler.java':[
154,
174,
],
'project:com/sun/org/apache/xerces/internal/xpointer/ElementSchemePointer.java':[
514,
527,
682,
708,
],
'project:com/sun/org/apache/xerces/internal/xpointer/ShortHandPointer.java':[
51,
],
'project:com/sun/org/apache/xerces/internal/xpointer/XPointerHandler.java':[
467,
481,
627,
656,
],
'project:com/sun/org/apache/xml/internal/dtm/DTMManager.java':[
355,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBase.java':[
1248,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseIterators.java':[
807,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDocumentImpl.java':[
147,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2RTFDTM.java':[
67,
],
'project:com/sun/org/apache/xml/internal/resolver/tools/ResolvingParser.java':[
105,
],
'project:com/sun/org/apache/xml/internal/resolver/tools/ResolvingXMLFilter.java':[
78,
],
'project:com/sun/org/apache/xml/internal/security/utils/I18n.java':[
41,
44,
52,
55,
58,
],
'project:com/sun/org/apache/xml/internal/serialize/DOMSerializerImpl.java':[
120,
],
'project:com/sun/org/apache/xml/internal/serialize/OutputFormat.java':[
196,
],
'project:com/sun/org/apache/xml/internal/serialize/XML11Serializer.java':[
126,
],
'project:com/sun/org/apache/xml/internal/serializer/ElemDesc.java':[
49,
61,
64,
70,
73,
112,
],
'project:com/sun/org/apache/xml/internal/serializer/OutputPropertiesFactory.java':[
191,
],
'project:com/sun/org/apache/xml/internal/serializer/SerializerFactory.java':[
75,
],
'project:com/sun/org/apache/xml/internal/serializer/ToUnknownStream.java':[
119,
124,
129,
134,
],
'project:com/sun/org/apache/xml/internal/serializer/utils/URI.java':[
149,
],
'project:com/sun/org/apache/xml/internal/utils/URI.java':[
146,
],
'project:com/sun/org/apache/xpath/internal/axes/FilterExprIterator.java':[
41,
42,
],
'project:com/sun/org/apache/xpath/internal/axes/FilterExprIteratorSimple.java':[
49,
50,
],
'project:com/sun/org/apache/xpath/internal/axes/FilterExprWalker.java':[
227,
],
'project:com/sun/org/apache/xpath/internal/compiler/FuncLoader.java':[
40,
],
'project:com/sun/org/apache/xpath/internal/compiler/OpCodes.java':[
633,
],
'project:com/sun/org/apache/xpath/internal/patterns/StepPattern.java':[
878,
],
'project:com/sun/security/auth/PolicyParser.java':[
104,
],
'project:com/sun/security/auth/module/JndiLoginModule.java':[
193,
],
'project:com/sun/security/auth/module/KeyStoreLoginModule.java':[
162,
],
'project:com/sun/security/auth/module/Krb5LoginModule.java':[
371,
],
'project:com/sun/security/auth/module/LdapLoginModule.java':[
370,
],
'project:com/sun/security/auth/module/NTLoginModule.java':[
66,
67,
68,
],
'project:com/sun/security/auth/module/SolarisLoginModule.java':[
60,
61,
62,
],
'project:com/sun/security/auth/module/UnixLoginModule.java':[
54,
55,
56,
],
'project:java/awt/CardLayout.java':[
118,
],
'project:java/awt/Choice.java':[
646,
],
'project:java/awt/List.java':[
1177,
],
'project:java/awt/TextComponent.java':[
114,
751,
],
'project:java/awt/color/ICC_Profile.java':[
1933,
],
'project:java/awt/font/NumericShaper.java':[
477,
478,
479,
480,
481,
482,
483,
484,
485,
486,
487,
488,
489,
491,
],
'project:java/awt/font/TextLayout.java':[
297,
],
'project:java/awt/image/ComponentColorModel.java':[
194,
],
'project:java/awt/image/DirectColorModel.java':[
114,
115,
116,
117,
118,
119,
120,
121,
],
'project:java/awt/image/LookupOp.java':[
79,
],
'project:java/beans/PropertyChangeSupport.java':[
490,
],
'project:java/beans/VetoableChangeSupport.java':[
479,
],
'project:java/io/FilePermission.java':[
818,
],
'project:java/io/SerializablePermission.java':[
99,
],
'project:java/lang/invoke/BoundMethodHandle.java':[
40,
41,
],
'project:java/lang/invoke/CallSite.java':[
92,
],
'project:java/lang/invoke/CountingMethodHandle.java':[
37,
],
'project:java/lang/invoke/MemberName.java':[
75,
],
'project:java/lang/invoke/MethodHandleImpl.java':[
50,
],
'project:java/lang/invoke/MethodType.java':[
826,
],
'project:java/lang/invoke/MethodTypeForm.java':[
49,
],
'project:java/lang/ref/Reference.java':[
95,
],
'project:java/math/BigDecimal.java':[
270,
],
'project:java/math/BigInteger.java':[
3064,
],
'project:java/math/MathContext.java':[
61,
],
'project:java/net/AbstractPlainDatagramSocketImpl.java':[
50,
53,
54,
55,
],
'project:java/net/SocketPermission.java':[
178,
1335,
],
'project:java/net/URI.java':[
2607,
2608,
2616,
2618,
],
'project:java/nio/channels/Channels.java':[
364,
439,
],
'project:java/security/BasicPermission.java':[
499,
],
'project:java/security/KeyFactory.java':[
98,
],
'project:java/security/Permissions.java':[
347,
554,
],
'project:java/security/SecureRandom.java':[
559,
563,
572,
576,
580,
],
'project:java/security/UnresolvedPermissionCollection.java':[
147,
],
'project:java/security/cert/CertPathBuilder.java':[
99,
],
'project:java/security/cert/CertPathValidator.java':[
99,
],
'project:java/text/AttributedString.java':[
741,
],
'project:java/text/BreakDictionary.java':[
89,
],
'project:java/text/CalendarBuilder.java':[
49,
],
'project:java/text/MessageFormat.java':[
1384,
1385,
1386,
1387,
],
'project:java/util/BitSet.java':[
83,
],
'project:java/util/EnumMap.java':[
119,
],
'project:java/util/GregorianCalendar.java':[
409,
],
'project:java/util/JapaneseImperialCalendar.java':[
105,
114,
],
'project:java/util/Locale.java':[
2026,
],
'project:java/util/PropertyPermission.java':[
619,
],
'project:java/util/Random.java':[
525,
],
'project:java/util/SimpleTimeZone.java':[
1127,
1129,
],
'project:java/util/Timer.java':[
110,
],
'project:java/util/concurrent/CopyOnWriteArrayList.java':[
1269,
],
'project:java/util/concurrent/SynchronousQueue.java':[
1149,
1151,
],
'project:java/util/concurrent/ThreadLocalRandom.java':[
86,
86,
86,
86,
86,
86,
86,
86,
],
'project:java/util/logging/LogManager.java':[
152,
],
'project:java/util/logging/SocketHandler.java':[
71,
],
'project:java/util/logging/StreamHandler.java':[
62,
],
'project:java/util/logging/XMLFormatter.java':[
47,
],
'project:java/util/zip/GZIPInputStream.java':[
149,
],
'project:javax/imageio/spi/ImageReaderSpi.java':[
104,
],
'project:javax/imageio/spi/ImageWriterSpi.java':[
106,
],
'project:javax/management/InvalidApplicationException.java':[
47,
],
'project:javax/management/Notification.java':[
111,
],
'project:javax/management/NumericValueExp.java':[
85,
],
'project:javax/management/ObjectName.java':[
316,
],
'project:javax/management/loading/MLet.java':[
212,
],
'project:javax/management/modelmbean/DescriptorSupport.java':[
118,
],
'project:javax/management/modelmbean/InvalidTargetObjectTypeException.java':[
83,
],
'project:javax/management/modelmbean/ModelMBeanAttributeInfo.java':[
150,
],
'project:javax/management/modelmbean/ModelMBeanConstructorInfo.java':[
123,
],
'project:javax/management/modelmbean/ModelMBeanInfoSupport.java':[
138,
],
'project:javax/management/modelmbean/ModelMBeanNotificationInfo.java':[
131,
],
'project:javax/management/modelmbean/ModelMBeanOperationInfo.java':[
143,
],
'project:javax/management/modelmbean/XMLParseException.java':[
79,
],
'project:javax/management/openmbean/CompositeDataInvocationHandler.java':[
247,
],
'project:javax/management/relation/MBeanServerNotificationFilter.java':[
107,
],
'project:javax/management/relation/RelationNotification.java':[
115,
],
'project:javax/management/relation/RelationTypeSupport.java':[
106,
],
'project:javax/management/relation/Role.java':[
89,
],
'project:javax/management/relation/RoleInfo.java':[
99,
],
'project:javax/management/relation/RoleResult.java':[
83,
],
'project:javax/management/relation/RoleUnresolved.java':[
92,
],
'project:javax/naming/NameImpl.java':[
60,
],
'project:javax/print/attribute/standard/MediaPrintableArea.java':[
85,
],
'project:javax/rmi/CORBA/ClassDesc.java':[
42,
47,
],
'project:javax/security/auth/PrivateCredentialPermission.java':[
120,
],
'project:javax/security/auth/Subject.java':[
972,
],
'project:javax/security/auth/kerberos/DelegationPermission.java':[
349,
],
'project:javax/security/auth/kerberos/ServicePermission.java':[
557,
],
'project:javax/sql/rowset/serial/SerialDatalink.java':[
60,
68,
],
'project:javax/sql/rowset/spi/SyncFactory.java':[
235,
254,
264,
268,
431,
],
'project:javax/swing/ImageIcon.java':[
75,
76,
],
'project:javax/swing/JComponent.java':[
257,
301,
329,
341,
342,
343,
344,
345,
346,
347,
],
'project:javax/swing/JFormattedTextField.java':[
254,
],
'project:javax/swing/JList.java':[
3190,
3191,
],
'project:javax/swing/JMenu.java':[
142,
157,
158,
],
'project:javax/swing/JMenuBar.java':[
102,
103,
104,
],
'project:javax/swing/JMenuItem.java':[
99,
100,
],
'project:javax/swing/JPopupMenu.java':[
130,
133,
134,
],
'project:javax/swing/MenuSelectionManager.java':[
43,
44,
],
'project:javax/swing/SwingUtilities.java':[
58,
59,
],
'project:javax/swing/TimerQueue.java':[
52,
],
'project:javax/swing/colorchooser/DefaultPreviewPanel.java':[
64,
],
'project:javax/swing/plaf/basic/BasicMenuItemUI.java':[
89,
91,
92,
],
'project:javax/swing/plaf/basic/BasicMenuUI.java':[
57,
60,
61,
62,
],
'project:javax/swing/plaf/basic/BasicPopupMenuUI.java':[
73,
74,
],
'project:javax/swing/plaf/basic/BasicProgressBarUI.java':[
1149,
],
'project:javax/swing/plaf/basic/BasicToolBarUI.java':[
58,
],
'project:javax/swing/plaf/metal/MetalFileChooserUI.java':[
82,
],
'project:javax/swing/plaf/metal/MetalHighContrastTheme.java':[
54,
],
'project:javax/swing/plaf/metal/MetalIconFactory.java':[
69,
70,
72,
73,
1880,
],
'project:javax/swing/plaf/metal/MetalInternalFrameUI.java':[
54,
],
'project:javax/swing/plaf/metal/MetalLookAndFeel.java':[
103,
],
'project:javax/swing/plaf/metal/MetalProgressBarUI.java':[
49,
50,
],
'project:javax/swing/plaf/nimbus/ArrowButtonPainter.java':[
49,
50,
51,
61,
],
'project:javax/swing/plaf/nimbus/ButtonPainter.java':[
58,
59,
61,
],
'project:javax/swing/plaf/nimbus/CheckBoxMenuItemPainter.java':[
54,
55,
67,
],
'project:javax/swing/plaf/nimbus/CheckBoxPainter.java':[
62,
64,
117,
],
'project:javax/swing/plaf/nimbus/ComboBoxArrowButtonPainter.java':[
60,
61,
62,
101,
],
'project:javax/swing/plaf/nimbus/ComboBoxPainter.java':[
62,
],
'project:javax/swing/plaf/nimbus/ComboBoxTextFieldPainter.java':[
48,
50,
51,
71,
],
'project:javax/swing/plaf/nimbus/DesktopIconPainter.java':[
46,
49,
62,
],
'project:javax/swing/plaf/nimbus/DesktopPanePainter.java':[
47,
48,
49,
71,
],
'project:javax/swing/plaf/nimbus/EditorPanePainter.java':[
48,
50,
51,
61,
],
'project:javax/swing/plaf/nimbus/FileChooserPainter.java':[
58,
131,
189,
],
'project:javax/swing/plaf/nimbus/FormattedTextFieldPainter.java':[
53,
54,
],
'project:javax/swing/plaf/nimbus/InternalFramePainter.java':[
50,
76,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePaneCloseButtonPainter.java':[
53,
55,
97,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePaneIconifyButtonPainter.java':[
52,
55,
100,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePaneMaximizeButtonPainter.java':[
62,
142,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePaneMenuButtonPainter.java':[
53,
55,
103,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePanePainter.java':[
46,
47,
48,
49,
57,
],
'project:javax/swing/plaf/nimbus/MenuBarMenuPainter.java':[
48,
50,
51,
60,
],
'project:javax/swing/plaf/nimbus/MenuBarPainter.java':[
47,
49,
50,
63,
],
'project:javax/swing/plaf/nimbus/MenuItemPainter.java':[
48,
50,
51,
60,
],
'project:javax/swing/plaf/nimbus/MenuPainter.java':[
53,
54,
66,
],
'project:javax/swing/plaf/nimbus/OptionPaneMessageAreaOptionPaneLabelPainter.java':[
46,
48,
49,
58,
],
'project:javax/swing/plaf/nimbus/OptionPaneMessageAreaPainter.java':[
46,
47,
48,
49,
57,
],
'project:javax/swing/plaf/nimbus/OptionPanePainter.java':[
52,
98,
],
'project:javax/swing/plaf/nimbus/PasswordFieldPainter.java':[
53,
54,
],
'project:javax/swing/plaf/nimbus/PopupMenuPainter.java':[
47,
49,
50,
61,
],
'project:javax/swing/plaf/nimbus/PopupMenuSeparatorPainter.java':[
46,
48,
49,
58,
],
'project:javax/swing/plaf/nimbus/ProgressBarPainter.java':[
55,
56,
114,
],
'project:javax/swing/plaf/nimbus/RadioButtonMenuItemPainter.java':[
57,
58,
70,
],
'project:javax/swing/plaf/nimbus/RadioButtonPainter.java':[
61,
62,
63,
103,
127,
],
'project:javax/swing/plaf/nimbus/ScrollBarButtonPainter.java':[
50,
51,
52,
91,
],
'project:javax/swing/plaf/nimbus/ScrollBarPainter.java':[
45,
46,
47,
48,
56,
],
'project:javax/swing/plaf/nimbus/ScrollBarThumbPainter.java':[
51,
52,
53,
86,
],
'project:javax/swing/plaf/nimbus/ScrollBarTrackPainter.java':[
49,
50,
74,
],
'project:javax/swing/plaf/nimbus/ScrollPanePainter.java':[
50,
51,
61,
],
'project:javax/swing/plaf/nimbus/SeparatorPainter.java':[
46,
48,
49,
58,
],
'project:javax/swing/plaf/nimbus/SliderPainter.java':[
45,
46,
47,
48,
56,
],
'project:javax/swing/plaf/nimbus/SliderThumbPainter.java':[
60,
61,
102,
],
'project:javax/swing/plaf/nimbus/SliderTrackPainter.java':[
47,
48,
50,
69,
],
'project:javax/swing/plaf/nimbus/SpinnerNextButtonPainter.java':[
61,
62,
99,
],
'project:javax/swing/plaf/nimbus/SpinnerPainter.java':[
46,
47,
48,
49,
57,
],
'project:javax/swing/plaf/nimbus/SpinnerPanelSpinnerFormattedTextFieldPainter.java':[
50,
52,
53,
76,
],
'project:javax/swing/plaf/nimbus/SpinnerPreviousButtonPainter.java':[
60,
61,
62,
100,
],
'project:javax/swing/plaf/nimbus/SplitPaneDividerPainter.java':[
49,
52,
75,
],
'project:javax/swing/plaf/nimbus/SplitPanePainter.java':[
46,
47,
48,
49,
57,
],
'project:javax/swing/plaf/nimbus/TabbedPanePainter.java':[
50,
51,
52,
53,
61,
],
'project:javax/swing/plaf/nimbus/TabbedPaneTabAreaPainter.java':[
49,
51,
52,
70,
],
'project:javax/swing/plaf/nimbus/TabbedPaneTabPainter.java':[
57,
58,
59,
104,
],
'project:javax/swing/plaf/nimbus/TableEditorPainter.java':[
51,
52,
62,
],
'project:javax/swing/plaf/nimbus/TableHeaderPainter.java':[
48,
49,
50,
60,
],
'project:javax/swing/plaf/nimbus/TableHeaderRendererPainter.java':[
55,
56,
88,
],
'project:javax/swing/plaf/nimbus/TextAreaPainter.java':[
55,
56,
],
'project:javax/swing/plaf/nimbus/TextFieldPainter.java':[
53,
54,
],
'project:javax/swing/plaf/nimbus/TextPanePainter.java':[
48,
50,
51,
61,
],
'project:javax/swing/plaf/nimbus/ToggleButtonPainter.java':[
59,
60,
62,
],
'project:javax/swing/plaf/nimbus/ToolBarButtonPainter.java':[
52,
54,
75,
],
'project:javax/swing/plaf/nimbus/ToolBarPainter.java':[
52,
53,
66,
],
'project:javax/swing/plaf/nimbus/ToolBarToggleButtonPainter.java':[
59,
61,
97,
],
'project:javax/swing/plaf/nimbus/ToolTipPainter.java':[
46,
48,
49,
59,
],
'project:javax/swing/plaf/nimbus/TreeCellEditorPainter.java':[
50,
51,
52,
62,
],
'project:javax/swing/plaf/nimbus/TreeCellPainter.java':[
51,
52,
62,
],
'project:javax/swing/plaf/nimbus/TreePainter.java':[
57,
58,
92,
],
'project:javax/swing/text/AbstractDocument.java':[
2950,
],
'project:javax/swing/text/StyleContext.java':[
748,
],
'project:javax/swing/text/html/FormView.java':[
881,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
657,
1365,
1431,
],
'project:javax/swing/text/html/HTMLWriter.java':[
55,
],
'project:javax/swing/text/html/parser/Parser.java':[
100,
],
'project:javax/xml/crypto/dsig/TransformService.java':[
282,
],
'project:javax/xml/datatype/Duration.java':[
104,
],
'project:javax/xml/parsers/DocumentBuilderFactory.java':[
44,
53,
],
'project:javax/xml/parsers/SAXParserFactory.java':[
46,
],
'project:javax/xml/soap/MessageFactory.java':[
69,
],
'project:javax/xml/transform/stax/StAXResult.java':[
72,
],
'project:org/xml/sax/helpers/NamespaceSupport.java':[
844,
],
'project:org/xml/sax/helpers/XMLFilterImpl.java':[
729,
],
}
